pkg/image/importer/credentials.go

```diff
@@ -139,16 +138,8 @@ func (s *SecretCredentialStore) init() credentialprovider.DockerKeyring {
 		}
 	}
 
-	secretsv1 := make([]kapiv1.Secret, len(s.secrets))
```

Comment:
round two... make the caller deal with conversion, honor callers that use external version (like registry).